pandas/tseries/tdi.py

```diff
@@ -353,36 +353,16 @@ def days(self):
         return self._get_field('days')
 
     @property
-    def hours(self):
-        """ The number of integer hours for each element """
-        return self._get_field('hours')
-
-    @property
-    def minutes(self):
-        """ The number of integer minutes for each element """
-        return self._get_field('minutes')
-
-    @property
     def seconds(self):
-        """ The number of integer seconds for each element """
+        """ The number of integer total seconds for each element """
```

Comment:
'total' is also not fully correct in the sense it is only a part of the timedelta expressed in seconds.

The docstring of `datetime.timedelta.seconds` says it as " Number of seconds (>= 0 and less than 1 day)."